vagrant/ansible/roles/samba.setup/tasks/cephfs/main.yml

```diff
@@ -0,0 +1,32 @@
+---
+- name: Install samba and samba-vfs-cephfs packages
+  yum:
+    name: samba-vfs-cephfs
+    state: present
+
+- name: Create and configure Samba share directory
+  run_once: true
+  block:
+    - name: Create Samba share directory
+      file:
+        path: "{{ item.cluster_volume }}"
+        state: directory
+        mode: 0777
+      loop: "{{ samba_shares }}"
+
+    - name: SELinux - Allow access to mount point
+      sefcontext:
+        target: "{{ item.cluster_volume }}"
+        setype: mnt_t
+        state: present
+      loop: "{{ samba_shares }}"
+
+    - name: SELinux - Apply changes
+      command: restorecon -R "{{ item.cluster_volume }}"
+      loop: "{{ samba_shares }}"
```

Comment:
We have the above tasks in a separate _fix_share_permissions.yml_ for GlusterFS and XFS. Shall we follow the same pattern?

Reply:
Since we only create one filesystem here (and it's unlikely we'll have more), we don't really need to separate those tasks.

Reply:
Well, on a high level it was the case previously too(especially with XFS) but I won't insist anymore.

Reply:
In the latest update, when a subvolume is created, it's possible to specify the mode for the root directory directly in the create command, so there's no need for a *fix_share_permissions.yml* now.